controller/backing_image_controller.go

```
@@ -310,22 +345,604 @@
+
+	readyV2CopyDiskUUIDs := []string{}
+	requireV2CopyDiskUUIDs := []string{}
+
+	for v2DiskUUID := range bi.Spec.DiskFileSpecMap {
+		biStatus, exists := bi.Status.DiskFileStatusMap[v2DiskUUID]
+		if exists {
+			if biStatus.State == longhorn.BackingImageStateReady {
+				readyV2CopyDiskUUIDs = append(readyV2CopyDiskUUIDs, v2DiskUUID)
+			}
+			if backingImageInProgress(biStatus.State) {
+				log.Infof("There is one v2 backing image copy in progress, prepare the backing image one at a time", bi.Name)
```

golangci-lint (1.62.2)

printf: (*github.com/sirupsen/logrus.Entry).Infof call has arguments but no formatting directives

(govet)